apps/frontend/src/components/modules/Auth/AuthBar/SignUpForm/SignUpForm.tsx

```diff
@@ -0,0 +1,134 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import clsx from 'clsx';
+import * as yup from 'yup';
+
+import Button from '../../../../ui/Button/Button';
+
+import styles from './SignUpForm.module.css';
+
+const schema = yup.object().shape({
```

Comment:
Use our zod schemas instead of yup. Sent you an example in a chat. Uninstall yup library